config/cesm/machines/config_machines.xml

```diff
@@ -1577,6 +1577,142 @@
      </environment_variables>
   </machine>
 
+  <machine MACH="eddi">
```

Comment:
This machine definition looks like it is intended for a single user - I think that it might be more appropriate to add it to the users .cime/config_machines.xml file instead of here.

Reply:
I asked Ryan, and it's his workstation that a couple people use. I've removed it and I'll point him to the use of .cime/config_machines.xml.